.github/workflows/check.yml

```diff
@@ -36,6 +51,7 @@ jobs:
             libasan8 libcmocka-dev libedit-dev libarchive-dev \
             libevent-dev libsmartcols-dev libnuma-dev python3-pyelftools \
             socat tcpdump traceroute graphviz iproute2 iputils-ping ndisc6
+          mkdir -p ~/.ccache
```

Comment:
Why is it needed?

Reply:
I thought it was after reading a warning from the cache action stating that the folder didn't exist and didn't cache anything.

Reply:
In any case, if this is really needed, it should go to the previous patch.